### PR TITLE
[MIG] github_connector_odoo: Migration to 12.0

### PR DESCRIPTION
**Pull request by cubells**
Create README using oca fragments

Standard migration

cc @Tecnativa

### CODE REVIEW

File: github_connector_odoo/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+* Once installed, go to your organization, and set extra settings:
+
+1. The name of your organization in the author keys of the manifest odoo
+   modules
+2. The URL of the file that contains IDs of your repositories for the runbot
+
+.. image:: /github_connector_odoo/static/description/github_organization_form.png
```
Comment (pedrobaeza) [inline github_connector_odoo/readme/CONFIGURE.rst]: This URL path is not correct. It should be relative as you can see here: https://github.com/OCA/maintainer-tools/blob/36453afeaed039f5f65e5cc4dd7910cb3235b811/tools/gen_addon_readme.py#L148

File: github_connector_odoo/readme/ROADMAP.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Possible improvements :
+
+* Implement deep code source analysis, like the website http://odoo-code-search.com/
+  and specially:
+
+1. Possibility to search by field or by model name. (Ex: field:invoice_id)
+2. Possibility to display the number of XML, Python, Yaml, HTML, CSS lines
+
+* Implement Social feature, like possibility to comment a module, add a
```
Comment (pedrobaeza) [inline github_connector_odoo/readme/ROADMAP.rst]: Remove the last 2 paragraphs, as they are covered now by other module.

File: github_connector_odoo/views/view_odoo_license.xml [hunk 3]
```diff
@@ -76,7 +76,7 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                     <t t-name="kanban-box">
                     <div class="oe_module_vignette oe_kanban_global_click">
                         <t t-if="record.image.raw_value">
-                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon"/>
+                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon" alt=""/>
```
Comment (pedrobaeza) [inline github_connector_odoo/views/view_odoo_license.xml]: Put an alt value, not empty.

File: github_connector_odoo/views/view_odoo_module.xml [hunk 4]
```diff
@@ -87,10 +87,10 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                     <t t-name="kanban-box">
                     <div class="oe_module_vignette oe_kanban_global_click">
                         <t t-if="record.image.raw_value">
-                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon"/>
+                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon" alt=""/>
```
Comment (pedrobaeza) [inline github_connector_odoo/views/view_odoo_module.xml]: not empty

File: github_connector_odoo/views/view_odoo_module.xml [hunk 5]
```diff
@@ -87,10 +87,10 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
...
                         </t>
                         <t t-if="!record.image.raw_value">
-                            <img t-att-src='_s + "/github_connector_odoo/static/src/img/default_image_module.png"' class="oe_module_icon"/>
+                            <img t-att-src='_s + "/github_connector_odoo/static/src/img/default_image_module.png"' class="oe_module_icon" alt=""/>
```
Comment (pedrobaeza) [inline github_connector_odoo/views/view_odoo_module.xml]: not empty

File: github_connector_odoo/views/view_odoo_module_version.xml [hunk 6]
```diff
@@ -111,10 +113,10 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                     <t t-name="kanban-box">
                     <div class="oe_module_vignette oe_kanban_global_click">
                         <t t-if="record.image.raw_value">
-                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon"/>
+                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon" alt=""/>
```
Comment (pedrobaeza) [inline github_connector_odoo/views/view_odoo_module_version.xml]: not empty

File: github_connector_odoo/views/view_odoo_module_version.xml [hunk 7]
```diff
@@ -111,10 +113,10 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
...
                         </t>
                         <t t-if="!record.image.raw_value">
-                            <img t-att-src='_s + "/github_connector_odoo/static/src/img/default_image_module_version.png"' class="oe_module_icon"/>
+                            <img t-att-src='_s + "/github_connector_odoo/static/src/img/default_image_module_version.png"' class="oe_module_icon" alt=""/>
```
Comment (pedrobaeza) [inline github_connector_odoo/views/view_odoo_module_version.xml]: not empty

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
You can squash a bit initial commit history by author.

**Comment by pedrobaeza** [comment]
@victormmtorres can you take this PR and fix my comments as @cubells is now busy with other things?

**Comment by martintorresvm** [comment]
@pedrobaeza please have a check

File: github_connector_odoo/readme/CONFIGURE.rst [hunk 8]
```diff
@@ -4,7 +4,7 @@
    modules
 2. The URL of the file that contains IDs of your repositories for the runbot
 
-.. image:: /github_connector_odoo/static/description/github_organization_form.png
+.. image:: ..static/description/github_organization_form.png
```
Comment (pedrobaeza) [inline github_connector_odoo/readme/CONFIGURE.rst]: `..` > `./`

File: github_connector_odoo/views/view_odoo_license.xml [hunk 9]
```diff
@@ -76,7 +76,7 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                     <t t-name="kanban-box">
                     <div class="oe_module_vignette oe_kanban_global_click">
                         <t t-if="record.image.raw_value">
-                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon" alt=""/>
+                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon" alt="${record.image.name}"/>
```
Comment (pedrobaeza) [inline github_connector_odoo/views/view_odoo_license.xml]: Not valid on QWeb. You have to use `t-att-alt="variable"`, but `record.image.name` is not a valid variable. I would say `record.name.raw_value` if the model has a field name and it's included in the view. `raw_value` is due to this being a JS QWeb view.

**Comment by martintorresvm** [comment]
Changes done @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Merging for continuing with the rest of the dependency chain.

**Pull request merged**